package-lock.json

```diff
@@ -1,8 +1,12971 @@
 {
   "name": "jellyfin-org",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
We probably want to keep Node 14/NPM 6 until Node 16 enters LTS (It's not there yet, I think it's for next month).

Reply:
On October 26

https://nodejs.org/en/about/releases/